src/main/java/org/spongepowered/api/command/CommandCallable.java

```diff
@@ -79,6 +79,20 @@
     boolean testPermission(CommandSource source);
 
     /**
+     * Test whether this command is allowed to be executed by the specified
```

Comment:
@simon816 I'm not familiar with _exactly_ the JavaDoc standards for tense but "Test" works grammatically. It's saying "this method will test" rather than "this method tests." However, this might be changed due to standards, not quite sure.